script/setup/install-runc

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+#
+# Builds and installs runc to /usr/local/go/bin based off
+# the commit defined in RUNC.md
+#
+set -eu -o pipefail
+
+RUNC_COMMIT=$(grep RUNC_COMMIT RUNC.md | cut -d "=" -f 2)
```

Comment:
I'm -1 for forcibly parsing machine-unreadable document. Why not use TOML/YAML?

Reply:
If we move the sha out from the docs when we have to duplicate it and remember to update both, or we can't document the supported version.

Reply:
We can just put `Please refer to foo.toml` to `RUNC.md`